src/TimingHardwareInterface.cpp

```diff
@@ -45,27 +48,29 @@ TimingHardwareInterface::TimingHardwareInterface()
 }
 
 void
-TimingHardwareInterface::configure_uhal(const nlohmann::json& obj)
+TimingHardwareInterface::configure_uhal()
 {
-  if (obj.contains("connections_file")) {
-    m_connections_file =  obj["connections_file"];
-  }
-  else
-  {
-    // throw error
-  }
+  // auto mdal = m_params->module<dal::TimingHardwareManagerPDIParameters>(get_name()); 
```

Comment:
@gcrone I think all this `m_uhal_log_level` code can be replaced by a simple: 
`auto mdal = m_params->module<dal::TimingHardwareManagerPDIParameters>(get_name());  ` 
`m_uhal_log_level = mdal->get_uhal_log_level();` 
but I can't get the necessary module into this file as I get the following error:
`/users/jr19588/fd-nightly2202/sourcecode/timinglibs/src/TimingHardwareInterface.cpp: In member function 'void dunedaq::timinglibs::TimingHardwareInterface::configure_uhal()':
/users/jr19588/fd-nightly2202/sourcecode/timinglibs/src/TimingHardwareInterface.cpp:56:73: error: 'get_name' was not declared in this scope; did you mean 'tzname'?
   56 |   auto mdal = m_params->module<dal::TimingHardwareManagerPDIParameters>(get_name());
      |                                                                         ^~~~~~~~
      |                                                                         tzname
`

Reply:
even though I've added this path: 
`#include "timinglibs/dal/TimingHardwareManagerPDIParameters.hpp"`

Reply:
I think that's because `TimingHardwareInterface` is not a plugin so doesn't inherit the from the `NamedObject` class. You should extract the info you need in `TimingHardwareManagerPDII` and pass it to `TimingHardwareInterface::configure_uhal` or pass the `mdal` pointer and let it extract the values.